gce/survival-training/README.md

```diff
@@ -0,0 +1,93 @@
+# GCE survival training
+
+Training jobs that survive GCE shutdowns
+
+- - -
+
+[Google Compute Engine](https://cloud.google.com/compute/docs/) (GCE) gives you direct access to the computational backbone of Google Cloud Platform. You can use GCE to create virtual machines provisioned with the resources of your choosing ([satisfying the constraints of any quotas that you define](https://cloud.google.com/compute/quotas)).
+
+This has particular value when it comes to training machine learning models, as this gives you access to resources beyond those available to you physically. For example, GCE will soon offer preemptible GPU instances. This means that, if you designed your trainer appropriately, your cost per training step could be a fraction of what it would be on a dedicated instance.
```

Comment:
if this goes with the launch, perhaps just say "GCE offers..." as opposed to "will soon offer".  (if only so that you don't have to come back and change this)

Reply:
+1